src/Resources/config/services/notification-processing.xml

```diff
@@ -0,0 +1,87 @@
+            <tag name="controller.service_arguments"/>
+        </service>
+        <service id="Adyen\Shopware\Entity\Notification\NotificationEntityDefinition">
+            <tag name="shopware.entity.definition" entity="adyen_notification"/>
+        </service>
+        <service id="Adyen\Shopware\Service\NotificationService">
+            <argument type="service" id="adyen_notification.repository"/>
+        </service>
+
+        <!--Scheduled Tasks-->
+        <service id="Adyen\Shopware\ScheduledTask\ProcessNotifications">
+            <tag name="shopware.scheduled.task"/>
+        </service>
+        <service id="Adyen\Shopware\ScheduledTask\ProcessNotificationsHandler">
+            <argument type="service" id="scheduled_task.repository"/>
+            <argument type="service" id="Adyen\Shopware\Service\NotificationService"/>
+            <argument type="service" id="Adyen\Shopware\Service\Repository\OrderRepository"/>
+            <argument type="service" id="Shopware\Core\Checkout\Order\Aggregate\OrderTransaction\OrderTransactionStateHandler"/>
+            <argument type="service" id="payment_method.repository"/>
+            <argument type="service" id="Shopware\Core\Framework\Plugin\Util\PluginIdProvider"/>
+            <tag name="messenger.message_handler"/>
+            <call method="setLogger">
+                <argument key="$logger" type="service" id="monolog.logger.adyen_notification"/>
+            </call>
+        </service>
+        <service id="Adyen\Shopware\ScheduledTask\ScheduleNotifications">
+            <tag name="shopware.scheduled.task"/>
+        </service>
+        <service id="Adyen\Shopware\ScheduledTask\ScheduleNotificationsHandler">
+            <argument type="service" id="scheduled_task.repository" />
+            <argument type="service" id="Adyen\Shopware\Service\NotificationService"/>
+            <tag name="messenger.message_handler"/>
+            <call method="setLogger">
+                <argument key="$logger" type="service" id="monolog.logger.adyen_notification"/>
+            </call>
+        </service>
+        <service id="Adyen\Shopware\ScheduledTask\FetchPaymentMethodLogos">
+            <tag name="shopware.scheduled.task"/>
+        </service>
+        <service id="Adyen\Shopware\ScheduledTask\FetchPaymentMethodLogosHandler">
+            <argument type="service" id="scheduled_task.repository"/>
+            <argument type="service" id="Adyen\Shopware\Service\ConfigurationService"/>
+            <argument type="service" id="Shopware\Core\Content\Media\MediaService"/>
+            <argument type="service" id="payment_method.repository"/>
+            <argument type="service" id="media.repository"/>
+            <argument>%shopware.media.enable_url_upload_feature%</argument>
+            <tag name="messenger.message_handler"/>
+            <call method="setLogger">
+                <argument key="$logger" type="service" id="monolog.logger.adyen_cron"/>
+            </call>
+        </service>
```

Comment:
Maybe the logo fetching services and cron belong in a different XML?

Reply:
My preference would have been to have this file dedicated to all scheduled tasks (scheduled-tasks.xml), and move the notification receiver to a more generic services xml

Reply:
What do you think @msilvagarcia?